pkg/pb/metainfo.proto

```diff
@@ -24,6 +31,72 @@ service Metainfo {
     rpc ProjectInfo(ProjectInfoRequest) returns (ProjectInfoResponse);
 }
 
+message BucketItem {
+    bytes             name = 1;
+    encryption.Cipher path_cipher = 2;
+    bytes             attribution_id = 3;
+
+    google.protobuf.Timestamp created_at = 4;
+
+    int64                           default_segment_size = 5;
+    pointerdb.RedundancyScheme      default_redundancy_scheme = 6;
+    encryption.EncryptionParameters default_encryption_parameters = 7;
+}
+
+message BucketListItem {
+    bytes             name = 1;
+    bytes             attribution_id = 2;
+
+    google.protobuf.Timestamp created_at = 3;
```

Comment:
use gogo to have pure `time.Time` `google.protobuf.Timestamp created_at = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];`